### DISCUSSION TITLE
RuntimeError when using AutoHotkey.py from game directory

### DISCUSSION THREAD

**Discussion by fantonhike** [Perlence/AutoHotkey.py#26] [General]
Hello,
apologies if this is not the right place for this question, but I've been struggling with this for a while now.

I'm trying to use the wrapper for a game made with Ren'Py. Normally it can use 3rd party modules from a folder called 'python-packages' in the game's directory (it works fine with the keyboard module, for example). But when I test anything inside the game, it returns with `RuntimeError: AHK interop is not available.`  And if I run the traceback, I get this: 
```
File "C:\Users\fanto\Desktop\renpy-8.1.0-sdk\renpy\compat\__init__.py", line 61, in <module>
    import future.standard_library
ModuleNotFoundError: No module named 'future'
```
I have tried updating and re-installing future, but the same message comes up every time.

Thanks

**Comment by Perlence** [comment]
Hey @fantonhike,

Unfortunately, you can't use AutoHotkey.py from a regular Python interpreter, it has to be embedded in an AutoHotkey process. This is why you see the RuntimeError, which also tells you how to run your script in a way that takes care of the embedding. It looks like this:

```
py -m ahkpy your_script.py
```

If you want to run Ren'Py as usual and interact with AutoHotkey in the same process, I would suggest using [spyoungtech/ahk](https://github.com/spyoungtech/ahk).

If you are okay running your AutoHotkey.py script in a separate process from Ren'Py, you can do just that.
